ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py

```diff
@@ -711,3 +711,53 @@ def test_paddle_rot90(
         k=k,
         axes=tuple(axes),
     )
+
+
+@st.composite
+def _chunk_helper(draw):
+    dtype, x, shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("float"),
+            min_num_dims=1,
+            ret_shape=True,
+        )
+    )
+    axis = draw(helpers.get_axis(shape=shape, force_int=True))
+    if shape[axis] == 0:
+        chunks = 0
+    else:
+        factors = []
+        for i in range(1, shape[axis] + 1):
+            if shape[axis] % i == 0:
+                factors.append(i)
+        chunks = draw(st.sampled_from(factors))
+    return dtype, x, axis, chunks
+
+
+# chunk
+@handle_frontend_test(
+    fn_tree="paddle.chunk",
+    x_dim_chunks=_chunk_helper(),
+    test_with_out=st.just(False),
+)
+def test_paddle_chunk(
+    *,
+    x_dim_chunks,
+    fn_tree,
+    on_device,
+    frontend,
+    test_flags,
+    backend_fw,
+):
+    dtype, x, axis, chunks = x_dim_chunks
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        backend_to_test=backend_fw,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=x[0],
```

Comment:
The tests are failing due to this unexpected argument `input`, you probably meant `x`.

Reply:
thanks @bipinKrishnan ..resolved